### PR TITLE
changes to links

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: about/quick_start.adoc [hunk 1]
```diff
@@ -4,37 +4,37 @@
 [#introduction]
 == Introduction
 
-See the product architecture at xref:../about/architecture.adoc[Architecture]. Review the xref:../about/components.adoc#components[Components] topics to learn more about what is installed with Red Hat Advanced Cluster Management for Kubernetes when you accept the license and subscription agreement.
+See the product architecture at xref:../about/architecture.adoc#multicluster-architecture[Architecture]. Review the xref:../about/components.adoc#components[Components] topics to learn more about what is installed with Red Hat Advanced Cluster Management for Kubernetes when you accept the license and subscription agreement.
```
Comment (dockerymick) [inline about/quick_start.adoc]: The page name doesnt match, change Architecture to Multicluster architecture

File: about/quick_start.adoc [hunk 2]
```diff
@@ -4,37 +4,37 @@
...
 
-After you learn about the hub cluster and managed cluster architecture, learn about the link:../manage_cluster#supported-clouds[Supported clouds], which lists the cloud provider cluster options.
+After you learn about the hub cluster and managed cluster architecture, learn about the link:../manage_cluster/supported-clouds.adoc#supported-clouds[Supported clouds], which lists the cloud provider cluster options.
```
Comment (dockerymick) [inline about/quick_start.adoc]: Broken link on line 9 

Change supported-clouds.adoc to supported_clouds.adoc

File: about/quick_start.adoc [hunk 3]
```diff
@@ -43,7 +43,7 @@ When you create a managed cluster, the new managed cluster imports automatically
 You can start managing applications on any created and imported managed clusters.
 The types of resources that you can create are applications, channels, subscriptions, and placement rules.
 
-. Learn more about the resources and how to create and manage them at link:../manage_applications[Managing applications].
+. Learn more about the resources and how to create and manage them at link:../manage_applications/app_management_overview#managing-applications[Managing applications].
```
Comment (dockerymick) [inline about/quick_start.adoc]: add the file extension `.adoc`

File: about/quick_start.adoc (hunk 1 above)
Comment (swopebe) [reply about/quick_start.adoc]: @cadawson you can commit to the PR you review to help eliminate steps with new PRs if you would like.

Comment (swopebe) [reply about/quick_start.adoc]: Why do we need this? #multicluster-architecture

File: about/quick_start.adoc (hunk 2 above)
Comment (swopebe) [reply about/quick_start.adoc]: Why do we need three references to the same thing in one link?

File: about/quick_start.adoc [hunk 4]
```diff
@@ -4,37 +4,37 @@
...
 
-If you experience problems, see the link:troubleshoot_acm/troubleshooting.adoc[Troubleshooting guide] to learn about the `mustgather` command and see documented troubleshooting tasks that might help resolve issues.
+If you experience problems, see the link:../troubleshoot_acm/troubleshooting.adoc[Troubleshooting guide] to learn about the `mustgather` command and see documented troubleshooting tasks that might help resolve issues.
 
 [#install]
 == Install
 
-. Before you install Red Hat Advanced Cluster Management for Kubernetes, review the system configuration requirements and settings at link:../install[Requirements and recommendations].
+. Before you install Red Hat Advanced Cluster Management for Kubernetes, review the system configuration requirements and settings at link:../install/requirements.adoc#requirement-and-recommendations[Requirements and recommendations].
 Get information about required operating systems and supported browsers.
```
Comment (swopebe) [inline about/quick_start.adoc:21]: Here, as well, seems redundant unless it is an anchor

File: about/quick_start.adoc (hunk 1 above)
Comment (swopebe) [reply about/quick_start.adoc]: tested in another PR with Chris, #these are not needed for links at least in the source, so we will see how they build. I thought those were only for anchors and not topics, but we will see.

**Pull request merged**